doc/Language/faq.pod6

```diff
@@ -362,18 +362,29 @@ multi postcircumfix:<[- ]> (Str:D $str is copy, +@indices) {
 say '0123456789'[- 1..3, 8 ];  # OUTPUT: «045679␤»
 =end code
 
-=head2 String: How can I split a string in equal parts
+=head2 String: How can I split a string in equal parts?
 
 L<.comb|/routine/comb> is accepting an optional L<Int|/type/Int>:
 
 =begin code
 .say for 'abcdefghijklmnopqrstuvwxyz'.comb: 8;  # OUTPUT: «abcdefgh␤ijklmnop␤qrstuvwx␤yz»
 =end code
 
+=head2 List: How can I show index while looping through a list?
```

Comment:
Unless I don't understand this properly, you've added everything back, not just this question. And again I'm sorry @treyharris , my main argument is that if it involves a single method it belongs better with that method, not in the FAQ. In [this case](https://docs.raku.org/routine/kv), the documentation for `.kv` is probably LTA, mostly repeating "it interleaves indexes and whatnot", so the way to proceed here is
1. Raise an issue about it, as in "the documentation for `.kv` does not mention that it can be used to loop through a list explicitly".
2. Talk it out to see where that improvement would belong better. As raised, it's probably the best to just change the method description, but it's described in like 6 classes, so where would it go better? Maybe in a `language` tutorial?
3. Propose a change where there's a consensus. 

"Single source of truth" also applies here. Methods, classes, are better described in the best possible way in a single place. Someone checking out the reference shouldn't be like "Oh, wait, there's more in this other page called *faq*". FAQs and tutorial are mostly devoted to patterns and actions involving stuff that can't possibly go in any individual page. It's not this case.